src/code/z_message_PAL.c

```diff
@@ -3002,14 +2999,19 @@ void Message_DrawDebugText(PlayState* play, Gfx** p) {
     *p = GfxPrint_Close(&printer);
     GfxPrint_Destroy(&printer);
 }
+#endif
 
 void Message_Draw(PlayState* play) {
     Gfx* plusOne;
     Gfx* polyOpaP;
+
+#if OOT_DEBUG
     s16 watchVar;
+#endif
```

Comment:
was hoping there was a way to scope this temp so it could be ifdeffed out together but I tried and could not do it